src/it/basic/build.xml

```diff
@@ -33,10 +35,20 @@
 
         <wlp:install-feature ref="testServer" name="mongodb-2.0" whenFileExists="ignore" acceptLicense="true" />
         <!-- install the same feature twice - should not cause an error -->
-        <wlp:install-feature ref="testServer" name="mongodb-2.0" whenFileExists="ignore" acceptLicense="true" />
-
+        <wlp:install-feature ref="testServer" whenFileExists="ignore" acceptLicense="true">
```

Comment:
Yes, both do the same command process (featureManager install _mongodb-2.0_ and expect a RC of 0 (Ok) or 22 (Already exist) )